Formula/spdx-sbom-generator.rb

```diff
@@ -0,0 +1,18 @@
+class SpdxSbomGenerator < Formula
+  desc "Support CI generation of SBOMs via golang tooling"
+  homepage "https://github.com/spdx/spdx-sbom-generator"
+  url "https://github.com/spdx/spdx-sbom-generator/archive/refs/tags/v0.0.13.tar.gz"
+  sha256 "7d088f136a53d1f608b1941362c568d78cc6279df9c1bdb3516de075cb7f10c3"
+  license any_of: ["Apache-2.0", "CC-BY-4.0"]
+
+  depends_on "go" => :build
+
+  def install
+    system "make", "build-mac"
+    bin.install "bin/spdx-sbom-generator"
+  end
+
+  test do
+    assert_match /Output Package Manager dependency on SPDX format.*/, shell_output("#{bin}/spdx-sbom-generator --help")
```

Comment:
```suggestion
    assert_match "Output Package Manager dependency on SPDX format", shell_output("#{bin}/spdx-sbom-generator --help")
```
Just skip the regex; you don't need it.

Reply:
Commit 82d0b07 should resolve this issue. Thank you for your assistance!